torchvision/prototype/transforms/_augment.py

```diff
@@ -107,19 +107,19 @@ def __init__(self, alpha: float, p: float = 0.5) -> None:
         self._dist = torch.distributions.Beta(torch.tensor([alpha]), torch.tensor([alpha]))
 
     def forward(self, *inputs: Any) -> Any:
-        if not (has_any(inputs, features.Image, features.is_simple_tensor) and has_any(inputs, features.OneHotLabel)):
-            raise TypeError(f"{type(self).__name__}() is only defined for tensor images and one-hot labels.")
+        if not (has_any(inputs, features.Image, features.Video, features.is_simple_tensor) and has_any(inputs, features.OneHotLabel)):
+            raise TypeError(f"{type(self).__name__}() is only defined for tensor images/videos and one-hot labels.")
         if has_any(inputs, PIL.Image.Image, features.BoundingBox, features.Mask, features.Label):
             raise TypeError(
                 f"{type(self).__name__}() does not support PIL images, bounding boxes, masks and plain labels."
             )
         return super().forward(*inputs)
 
     def _mixup_onehotlabel(self, inpt: features.OneHotLabel, lam: float) -> features.OneHotLabel:
-        if inpt.ndim < 2:
+        if inpt.ndim != 2:
```

Comment:
I think it's better to specify the exact expected dimensions. I don't see any use-case where we expect anything other than 2.